### DISCUSSION TITLE
Mount() maybe something strangeness

### DISCUSSION THREAD

**Discussion by sueboy19** [livewire/livewire#6790] [1. Help]
OS: windows 10
PHP: 64bit 8.2.8
Laravel:  v10.23.0
Livewire: v3.0.3

1. Same code Two Developer A、B get different result. 
    A is ok, B is 404 Not Found /update on wire.model be changed. 
    Nginx same config. 
    B being developer is OK. After few day get 404. 
    Code is same from git and same commit.

2. B try other Page of this project, wire:click is OK. Only this page get problem. A is all ok.

3. mount() run finish, everything is correct. B try to wire.model change value then 404 Not Found /update.

Code like under
```php
public function mount()
{
    $this->asset_subjects = 
           Subject::select('......')->get();   // 404 Not Found /update

           DB::table(subject')::select('......')->get();  // OK
}
```
Subject::select('......')->get();  even limit(1) still get 404.
Check B run project, can sure view finish. Console log is No error. 

**strangeness**

**Comment by GabrielCrespoBrito** [comment]
Same error, it seems to happen only when the components use the **mount** or **boot** method
